components/OutboundLink/__tests__/OutboundLink.test.js

```diff
@@ -14,7 +13,7 @@ describe('OutboundLink', () => {
   });
 
   it('should render SVG when `hasIcon` is true', () => {
-    const wrapperWithIcon = mount(
+    const wrapperWithIcon = render(
```

Comment:
can you change this to `component`?

Reply:
Sorry, what are you wanting changed here? (new to React testing library - couldn't find a component method)

Reply:
Just the variable name. You can always just do:

`const component = render()`

or

`const { container } = render()` if you only need the container.